src/packagedcode/podspec.py

```diff
@@ -0,0 +1,271 @@
+            stripped_version = version.replace("'", "")
+            stripped_version = stripped_version.replace('"', "")
+            stripped_version = stripped_version.replace('{', "")
+            stripped_version = stripped_version.replace('}', "")
+            stripped_version = stripped_version.strip()
+            podspec_data['version'] = stripped_version
+        elif '.author' in line:
+            authors = re.sub(r"/*.*author.*= ", "", line)
+            stripped_authors = authors.replace("'", "")
+            stripped_authors = stripped_authors.replace('"', "")
+            stripped_authors = stripped_authors.replace('{', "")
+            stripped_authors = stripped_authors.replace('}', "")
+            stripped_authors = stripped_authors.strip()
+            stripped_authors = stripped_authors.split(', ')
+            author_names = []
+            author_email = []
+            for split_author in stripped_authors:
+                split_author = split_author.split('=>')
+                author = split_author[0].strip()
+                email = split_author[1].strip()
+                author_names.append(author)
+                author_email.append(email)
+            podspec_data['author'] = author_names
+            podspec_data['email'] = author_email
+        elif '.summary' in line:
+            summary = re.sub(r"/*.*summary.*= ", "", line)
+            stripped_summary = summary.replace("'", "")
+            stripped_summary = stripped_summary.replace('"', "")
+            stripped_summary = stripped_summary.replace('{', "")
+            stripped_summary = stripped_summary.replace('}', "")
+            stripped_summary = stripped_summary.strip()
+            podspec_data['summary'] = stripped_summary
+        elif '.description' in line:
+            podspec_data['description'] = get_description(location)
+        elif '.homepage' in line:
+            homepage_url = re.sub(r"/*.*homepage.*= ", "", line)
+            stripped_homepage_url = homepage_url.replace("'", "")
+            stripped_homepage_url = stripped_homepage_url.replace('"', "")
+            stripped_homepage_url = stripped_homepage_url.replace('%q{', "")
+            stripped_homepage_url = stripped_homepage_url.replace('}', "")
+            stripped_homepage_url = stripped_homepage_url.strip()
+            podspec_data['homepage_url'] = stripped_homepage_url
+        elif '.dependency' in line:
+            match = add_dependency_regex.match(line)
+            line = match.group('line')
+            podspec_data['dependencies'] = parse_line(line)
+        
+    deps = OrderedDict()
+    try:
+        if podspec_data['dependencies']:
+            dependency = podspec_data['dependencies']
+            for dep in dependency:
+                dep_name = dep[0]
+                dep.pop(0)
+                dep_version = dep
+                deps[dep_name] = dep_version
+    except KeyError:
+        pass
```

Comment:
Do we need an try statement here? why not use `.get()`?

Reply:
solved this with `.get()` :)